src/ts/createTyperighterPlugin.ts

```diff
@@ -106,7 +114,10 @@ const createTyperighterPlugin = <TMatch extends IMatch>(
     },
     props: {
       decorations: state => {
-        return plugin.getState(state).decorations;
+        const pluginState = plugin.getState(state);
+        return pluginState.config.isActive
+          ? pluginState.decorations
+          : emptyDecorationSet;
```

Comment:
This is where we decided to show/hide our document decorations – [`props.decorations`](https://prosemirror.net/docs/ref/#view.EditorProps.decorations) is a function which maps the stored plugin decorations to the decorations seen in the document.